src/components/ActiveFormList.js

```diff
@@ -29,46 +30,14 @@ class ActiveFormList extends Component {
     }
 
     fetchCases() {
-        let self = this;
         this.service.getMyCases({}, (data) => {
-            if (data && data.cases && data.cases.length > 0) {
-                this.setState({ 
-                    cases:data.cases.slice(0, 5).map(function(item) { 
-                        return {
-                            id:item.id,
-                            parties: self.parties(item.data),
-                            status: item.status,
-                            modified: item.modified,
-                            data: item.data
-                        };
-                    }),
-                    displayMyCasesEmptyLabel: false 
-                });
-            } else {
-                this.setState({
-                    cases:[],
-                    displayMyCasesEmptyLabel: true
-                })
-            }
+            let cases = renderCases(data.cases.slice(0, 5));
```

Comment:
can we use a different word than "render" cases?  I was thinking on first glance it would do something like the react render().  How about formatCases?  Or getCasesMap?  or mapCases?